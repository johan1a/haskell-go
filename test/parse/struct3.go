package main
import "fmt";

type L struct {
    x int; 
    k int;
}
