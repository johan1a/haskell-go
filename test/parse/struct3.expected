SourceFile (Package "main") [TopLevelDecl1 (TypeDecl [TypeSpec "L" (TypeLit (StructTypeLit (StructType1 [FieldDecl1 [IdDecl "x"] (TypeName (TypeNameIdentifier "int")),FieldDecl1 [IdDecl "k"] (TypeName (TypeNameIdentifier "int"))])))])]
